man/eza.1.md

```diff
@@ -302,6 +314,9 @@ For more information on the format of these environment variables, see the [eza_
 
 Overrides any `--git` or `--git-repos` argument
 
+## `EZA_MIN_LUMINANCE`
+Specifies the minimum luminance to use when decay is active. It's value can be between -100 to 100.
```

Comment:
I think it would be useful to have an example of use of this, qnd whqr is means to set values like -100, -50, 0, 23, 100